Topics/Development_Process/Standards_For_Best_Practices.md

```diff
@@ -0,0 +1,47 @@
+# Utilizing Standards to Adopt Best-Practices in Software Engineering
+
+## Table of Contents
+
+### [What Is a Standard?](#what-is-a-standard-1)
+
+### [Why Are Standards Important?](#why-are-standards-important-1)
+
+### [How Are Standards Created?](#how-are-standards-created-1)
```

Comment:
Overall, well done with you explanation! It was a very clear and concise view of how standards are created.

Reply:
Thank you very much 😇